app/poros/github_repo.rb

```diff
@@ -0,0 +1,11 @@
+class GithubRepo
+  def initialize
+    @repo_prs = GithubService.num_pull_requests
+  end
+
+  def num_merged_prs
+    @repo_prs.find_all do |pr|
```

Comment:
Count enum would work too 👍

Reply:
Indeed, better choice. Ill tweak that in my refactor work tomorrow - thanks for the recommendation!